kernel-modules/probe/collector_probe.h

```diff
@@ -0,0 +1,283 @@
+#ifndef __COLLECTOR_PROBE_H
+#define __COLLECTOR_PROBE_H
+
+/* The structs below correspond to the syscall tracepoint format found 
+ * in /sys/kernel/tracing/events/syscalls
+ * example: cat /sys/kernel/tracing/events/syscalls/sys_enter_setgid/format
```

Comment:
To be clear, are these manually written based on the `format` files or is there a tool that can be used to generate the structures based on those files?

Reply:
@Stringy those are manually written. I looked around some tooling for this but did not find anything useful. However, it looks like that [tplist](https://github.com/iovisor/bcc/blob/master/tools/tplist_example.txt) might help in future.

Reply:
Can we please update this comment to make it clear that this is the way these have been constructed

Reply:
resolved in https://github.com/kcrane/collector/commit/04a208802a9bec1159e32c31006af01b61de8bd5